src/github.com/getlantern/analytics/analytics.go

```diff
@@ -68,8 +68,21 @@ type Payload struct {
 	Event *Event
 }
 
+func Configure(trackingId string, version string, proxyAddr string) {
+	var err error
+	go func() {
+		httpClient, err = util.HTTPClient("", proxyAddr)
+		if err != nil {
+			log.Errorf("Could not create HTTP client via %s: %s", proxyAddr, err)
+			return
+		}
+		// Store new session info whenever client proxy is ready
+		sessionEvent(version, trackingId)
```

Comment:
It looks like this will now record a session event every time regardless of whether or not the user has chosen to auto-report stats @atavism.